back-end/__test__/utils.test.js

```diff
@@ -486,3 +487,139 @@ describe("Tests for compareMoney", () => {
         expect(compareMoney(a, b)).toBeGreaterThan(0);
     });
 });
+
+describe("generateTableFunction", () => {
+  it("should generate correct team stats for a single match", () => {
+    const completedMatches = {
+      rows: [
+        {
+          team1_id: 1,
+          team2_id: 2,
+          team1_score: 2,
+          team2_score: 1,
+        },
+      ],
+    };
+
+    const teamStats = generateTableFunction(completedMatches);
```

Comment:
generateTableFunction here is not imported as a function from the start (There is generatLeagueTable imported at the start instead which isnt being used) ---> might need to look at this

Reply:
changed function name in test to match import which is the correct name